airbyte-integrations/connectors/source-chargebee/acceptance-test-config.yml

```diff
@@ -5,64 +5,65 @@ test_strictness_level: high
-        status: "failed"
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
   discovery:
     tests:
-      - config_path: "secrets/config.json"
-        backward_compatibility_tests_config:
-          disable_for_version: "0.2.1" # New streams were added; fixed fields type
+    - config_path: "secrets/config.json"
+      backward_compatibility_tests_config:
+        disable_for_version: "0.2.1"   # New streams were added; fixed fields type
   basic_read:
     tests:
-      - config_path: "secrets/config.json"
-        timeout_seconds: 1200
-        empty_streams:
-          - name: "addon"
-            bypass_reason: "Not permitted for this site"
-          - name: "plan"
-            bypass_reason: "Not permitted for this site"
-          - name: "virtual_bank_account"
-            bypass_reason: "Cannot populate with test data"
-          - name: "subscription"
-            bypass_reason: "Unstable data. Field current_term_start updated daily"
-          - name: "customer"
-            bypass_reason: "Unstable data. Depends on subscription"
-          - name: "invoice"
-            bypass_reason: "Unstable data. Depends on subscription"
-          - name: "credit_note"
-            bypass_reason: "Unstable data. Depends on subscription"
-          - name: "event"
-            bypass_reason: "Unstable data. Depends on subscription"
-          - name: "unbilled_charge"
-            bypass_reason: "Empty stream. Unstable data"
-          - name: "hosted_page"
-            bypass_reason: "Empty stream. Unstable data"
-        expect_records:
-          path: "integration_tests/expected_records.jsonl"
-          extra_fields: no
-          exact_order: no
-          extra_records: yes
+    - config_path: "secrets/config.json"
+      timeout_seconds: 1200
+      empty_streams:
+      - name: "addon"
+        bypass_reason: "Not permitted for this site"
+      - name: "plan"
+        bypass_reason: "Not permitted for this site"
+      - name: "virtual_bank_account"
+        bypass_reason: "Cannot populate with test data"
+      - name: "subscription"
+        bypass_reason: "Unstable data. Field current_term_start updated daily"
+      - name: "customer"
+        bypass_reason: "Unstable data. Depends on subscription"
+      - name: "invoice"
+        bypass_reason: "Unstable data. Depends on subscription"
+      - name: "credit_note"
+        bypass_reason: "Unstable data. Depends on subscription"
+      - name: "event"
+        bypass_reason: "Unstable data. Depends on subscription"
+      - name: "unbilled_charge"
+        bypass_reason: "Empty stream. Unstable data"
+      - name: "hosted_page"
+        bypass_reason: "Empty stream. Unstable data"
+      expect_records:
+        path: "integration_tests/expected_records.jsonl"
+        extra_fields: no
+        exact_order: no
+        extra_records: yes
+      fail_on_extra_columns: false
```

Comment:
Despite intentation changes courtesy of ruamel.yaml, this line is the only addition in each config (`Hide whitespace` in the diff helps a lot with this!)